HDFOutputer.cc

```diff
@@ -105,6 +124,16 @@ get_prods_and_sizes(std::vector<product_t> & input,
 void 
 HDFOutputer::output(EventIdentifier const& iEventID, 
                     std::vector<SerializerWrapper> const& iSerializers) {
+#ifdef H5_TIMING_ENABLE
+  size_t total_data_size = 0;
+#endif
+  char *p = getenv("HEP_IO_TYPE");
```

Comment:
Pleas use the configuration system to get the values, not environment variables.